vendor/README.md

```diff
@@ -5,4 +5,5 @@ rm -rf safestringlib
 git clone https://github.com/intel/safestringlib
 rm -rf safestringlib/{.git,unittests}
 git add safestringlib/
+# Redo the changes done in https://github.com/citusdata/citus/pull/3541
```

Comment:
I think it'd be great if the required code-changes are in a single commit, and we comment here to cherry-pick that commit from this PR? Otherwise, it'd be painful in the future

Reply:
Done